firefox-ios/Client/Frontend/Browser/ToastType.swift

```diff
@@ -44,7 +43,7 @@ enum ToastType: Equatable {
         case .copyURL, .addBookmark: return nil
         }
 
-        return TabPanelViewAction(panelType: panelType,
+        return TabPanelViewAction(panelType: nil,
```

Comment:
Just curious more than anything, but why did you choose to pass back an action with `nil` panelType instead of returning `nil` here?

Reply:
In the TabTrayViewController, we need to get a redux action in a few places for a given ToastType. We still want to fire actions (undoClose, undoCloseAllTabs, etc.), but in this case the panelType context didn't seem important.

That said, I better check this on Monday, because I just tried deleting `panelType` from `TabPanelViewAction` and noticed it's used in a few places in `TabManagerMiddleware`... I have a feeling it might be superfluous though if we can get the isPrivate info elsewhere. 🤔

Reply:
FYI I looked closer and confirmed that none of the middlewares/reducers care about the panelType for any actions fired by a ToastType. 🤔  I've added a comment just for a bit more clarity.